src/main/java/Serf.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+public class Serf {
+    public static void main(String[] args) {
+        String VERTICALLINES = "    ____________________________________________________________";
```

Comment:
Should the naming of constant variables be separated by "_" between each word?
e.g. VERTICAL_LINES instead of VERTICALLINES